stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/OTPElementUI.kt

```diff
@@ -139,7 +142,10 @@ fun OTPElementUI(
                     },
                     modifier = textFieldModifier,
                     enabled = enabled,
-                    textStyle = MaterialTheme.typography.h2.copy(
+                    textStyle = TextStyle(
```

Comment:
This should not be referencing `MaterialTheme.typography.h2` because:

- `StripeTypography` does not use h2. If using `StripeTheme`, OTP component would use the default MaterialTypography#H2 (60sp), way larger than the OTP input boxes.
- We don't want this TextStyle size / weight to be customizable (color still relies on `stripeColors`) - the OTP input boxes are designed for a `24sp` text size.